pyo3-build-config/src/impl_.rs

```diff
@@ -826,7 +920,7 @@ fn ends_with(entry: &DirEntry, pat: &str) -> bool {
 /// So we must find the file in the following possible locations:
 ///
 /// ```sh
-/// # distribution from package manager, lib_dir should include lib/
+/// # distribution from package manager, (lib_dir may or may not include lib/)
```

Comment:
Hopefully this is okay - see implementation below. This was requested in Maturin to support cross-compiling with PyPy, where the `_sysconfigdata*.py` is not under the `lib` dir.